app/assets/stylesheets/partials/_navbar.sass

```diff
@@ -2,14 +2,14 @@ nav#navigation
   @extend .navbar, .navbar-inverse, .navbar-static-top
 
   a.nav-dropdown-img-toggle
-    @extend .dropdown-toggle, .hidden-xs, .btn, .navbar-btn
```

Comment:
I'm not 100% sure why I extended so many classes... Guess it had something to do with how I wanted the collapsed, mobile menu to look like, it has a slightly different structure then the normal one.